Felienne/hedy
# Implement customizable classes

Provide a general summary of your changes in the Title of the PR

**Description**
In this PR we implement customizable classes. Enabling teachers to select specific adventures, hide example programs and/or hide levels. This feature will allow teachers to customize their teaching and restrict students to specific levels and/or adventures helping the teaching.

**Fix for**
This PR fixed #1299.

**How to test**
Log in as a teacher account and navigate to a class. Choose the "customize class" button and select preferences for each level. Then, on a student account (enrolled to this class) navigate to the levels for which preferences where selected and notice the created restrictions.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
